desk/app/forums.hoon

```diff
@@ -49,23 +51,23 @@
     =^  cards  pub-forums  (give:du-forums [%forums %updates p.act ~] [bowl act])
     ::  Prints pub state so that we can observe change caused by poke,
     ::  Comment out line below when releasing.
-    ~&  >>>  read:du-forums
```

Comment:
I mentioned this in our group chat, but note that `read:du-forums` is only way for a host to obtain the latest version of its hosted boards. `pub-forums` doesn't hold that information. We probably want to call `read:du-forums` after every poke and store the result so that we won't have to call it again when a scry comes in.

Reply:
Yes, we'd have to cached the state either on-poke or on-scry. I'm hoping to talk to ~wicrum-wicrun about best practices here.